examples/ChatQnA/ChatQnA_Guide.rst

```diff
@@ -4,65 +4,80 @@ ChatQnA Sample Guide
 ####################
 
 Introduction/Purpose
-*********************
+********************
+
+Tom to provide. 
 
 Overview/Intro
 ==============
 
 Purpose
 =======
 
-AI Catalog Preview 
-==================
+Preview 
+=======
 
-(if applicable)
+AI catalog if applicable, or recorded demos. 
 
 Key Implementation Details 
 ==========================
 
-Tech overview
+Tech Overview
 *************
 
-How it works
+How It Works
 ============
 
+High level graphics to summarize the application.
+
 Expected Output
 ===============
 
-Customization
-==============
-
 Validation Matrix and Prerequisites
 ***********************************
 
 Architecture
 ************
 
+Includes microservice level graphics.
+
 Need to include the architecture with microservices. Like the ones Xigui/Chun made and explain in a para or 2 on the highlights of the arch including Gateway, UI, mega service, how models are deployed and how the microservices use the deployment service. The architecture can be laid out as general as possible, maybe  calling out “for e.g” on variable pieces. Will also be good to include a linw or 2 on what the overall use case is. For e.g. This chatqna is setup to assist in ansewering question on OPEA. The microservices are set up with RAG and llm pipeline to query on OPEA pdf documents 
 
-Microservice outline and diagram
+Microservice Outline and Diagram
 ================================
 
 Deployment
 **********
 
-.. tabs::
-
-   .. tab:: Single Node deployment
-
-      (IDC or on prem metal: Xeon, Gaudi, AI PC, Nvidia?)
-
-   .. tab:: Kubernetes 
-
-      K8S for Clusters.
-
-   .. tab:: Cloud Deployment
-
-      AWS and Azure.
-
-   .. tab:: Managed Services Deployment
-
-      Such as..
++--------------------------------------------------------------------------------------+
+| Single Node                                                                          |
+|                                                                                      |
++============================================+=========================================+
+| XEON Scalable Processors                   |Gaudi Servers                            |
+|                                            |                                         |
++--------------------------------------------+-----------------------------------------+
+| NNIDIA GPUs                                | AI PC                                   |
```

Comment:
Looks like a typo NNIDIA /NVIDIA?

Reply:
There are lots of typos in this place holder doc that will be updated later :)